modules/otel-config-instrumentation.adoc

```diff
@@ -142,6 +174,13 @@ instrumentation.opentelemetry.io/inject-apache-httpd: "true"
 [id="otel-configuration-of-dotnet-auto-instrumentation_{context}"]
 == Configuration of the .NET auto-instrumentation
 
+include::snippets/technology-preview.adoc[leveloffset=+1]
```

Comment:
@pavolloffay, I see several of these `include` `technology-preview` statements added in this PR.
To me, this PR looks like about instrumentation GA, which generally means removing, not adding, `technology-preview` admonitions.
Is this some other change included as part of this PR? Like we just realized that we should have added `technology-preview` here previously?
Let me know, and I'll take care of it.

Reply:
Only the SDK injection is GA therefore I have added tech-preview to the rest of the sections

Reply:
Before the TP banner was for the whole document, now as SDK injection is GA the TP has to move to each other injection sections